src/FsAutoComplete.Core/Lint.fs

```diff
@@ -27,6 +30,7 @@ let loadConfiguration (workspaceRoot: string option) (lintConfigFileRelativePath
   | None -> ConfigurationParam.Default
 
 let lintWithConfiguration (lintConfig: ConfigurationParam) ctok ast sourceCode typeCheckResults =
+  try
```

Comment:
this try-catch isn't strictly speaking necessary from a bug report or anything, but I want to protect against the call crashing in any way, taking down the entire FSAC process.

Reply:
👍